behave/README.md

```diff
@@ -22,6 +22,29 @@ to build locally:
 bash into the container and run them from there:
 `docker-compose run --entrypoint /bin/bash cla-end-to-end`
 
+## Current state of affiars for M1 (Arm64) users
+
+Apples new line of computers no longer use Intel processors. Instead Apple has made their own, this isn't curretly widly support (at the time of writting this). 
```

Comment:
typos :)
"this isn't currently widely supported (at the time of writing this). "